test/unit/Runners/PHPUnit/ExecutableTestTest.php

```diff
@@ -31,6 +33,26 @@ public function testGetCommandStringIncludesOptions()
         $this->assertEquals("'/usr/bin/phpunit' '--bootstrap' 'test/bootstrap.php' 'ClassNameTest' 'pathToFile'", $command);
     }
 
+    public function testGetCommandStringIncludesPassthruOptions()
+    {
+        $options = ['bootstrap' => 'test/bootstrap.php'];
+        $binary = '/usr/bin/phpunit';
+        $passthru = "'--prepend' 'xdebug-filter.php'";
+        $passthruPhp = "'-d' 'zend_extension=xdebug.so'";
+
+        $command = $this->call($this->executableTestChild, 'getFullCommandlineString', $binary, $options, $passthru, $passthruPhp);
+        // Note:
+        // '--log-junit' '/tmp/PT_LKnfzA'
+        // is appended by default where PT_LKnfzA is randomly generated - so we remove it from the resulting command
+        $command = preg_replace("# '--log-junit' '[^']+?'#", '', $command);
```

Comment:
OK nice, very strict way to test the result. 👍 (Personally I guess I would have just used `assertContains` here.)

Reply:
I would actually like to make the `'--log-junit' '/tmp/PT_LKnfzA` part configurable in `ExecutableTest` but that would probably be a bigger refactoring :)

As for "strictness": I try to avoid "contains" as it opens up the assertion for "unwanted" side effects + in this case I would need two assertions instead of one.